Services/persistence.js

```diff
@@ -0,0 +1,44 @@
+import { local } from "d3-selection";
+
+export const getUserTrack= (userId: string): any => {
+    // return dummy data for now
+    if (window.localStorage && "milestoneTrack" in window.localStorage) {
+        return JSON.parse(window.localStorage.getItem("milestoneTrack"));
+    }
+
+    return {
+        title: 'Software Developer',
+        milestoneByTrack: {
```

Comment:
Could this read from the constants instead and just assign random numbers between 0 and 5?